cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/internal/mazha/CouchClient.java

```diff
@@ -311,12 +311,39 @@ public ChangesResult changes(String filterName, Map<String, String> filterParame
         return this.changes(options);
     }
 
+    public ChangesResult changes(String selector, Object since, Integer limit) {
+        Map<String, Object> options = getDefaultChangeFeedOptions();
```

Comment:
There's a lot of copy and paste between this and the existing constructor, maybe hive this all off to a utility function which builds the options?